engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/instrument/RuntimeServerTesting.java

```diff
@@ -0,0 +1,93 @@
+package org.enso.interpreter.test.instrument;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.UUID;
+import org.enso.interpreter.test.Metadata;
+import org.enso.interpreter.test.instrument.RuntimeServerTest.TestContext;
+import org.enso.polyglot.runtime.Runtime$Api$CreateContextRequest;
+import org.enso.polyglot.runtime.Runtime$Api$CreateContextResponse;
+import org.enso.polyglot.runtime.Runtime$Api$MethodPointer;
+import org.enso.polyglot.runtime.Runtime$Api$OpenFileRequest;
+import org.enso.polyglot.runtime.Runtime$Api$OpenFileResponse$;
+import org.enso.polyglot.runtime.Runtime$Api$PushContextRequest;
+import org.enso.polyglot.runtime.Runtime$Api$Request;
+import org.enso.polyglot.runtime.Runtime$Api$Response;
+import org.enso.polyglot.runtime.Runtime$Api$StackItem$ExplicitCall;
+import org.enso.scala.wrapper.ScalaConversions;
+import scala.Option;
+
+/**
+ * Invoked from {@link RuntimeServerTest}. Reuses the same infrastructure while it allows writing
+ * new Java instead of in Scala.
+ */
+final class RuntimeServerTesting {
+  static void accessRuntimeCache(TestContext context) {
+
+    var contextId = UUID.randomUUID();
+    var requestId = UUID.randomUUID();
+    var reqOptId = Option.apply(requestId);
+    var moduleName = "Enso_Test.Test.Main";
+
+    var metadata = new Metadata("");
+    var id_x_0 = metadata.addItem(166, 1, "aa");
+    var id_x_1 = metadata.addItem(178, 1, "ab");
+
+    var code =
+        """
+      from Standard.Base.Runtime import value_for_uuid
+      from Standard.Base.Data.Numbers import all
+      import Standard.Base.IO
+
+      private v n = value_for_uuid n
+
+      main =
+          x_0 = 6
+          x_1 = 7
+          x_2 = (v "${aa}")*(v "${bb}")
```

Comment:
There seems to be no test for `Runtime.value_for_uuid`. Adding one in 5f0c43f0109fc2b8cf53899a6be91d687b29803a, CCing @4e6